_includes/v3.1/reqs-sys.md

```diff
@@ -37,9 +37,13 @@ to connect to, we provide instructions in the [installation documentation](./ins
 | BGP     | TCP      | 179  |
 | IPIP\*  | 4        | n/a  |
 
-\* Our manifests enable IPIP by default. If you disable IPIP, you won't need to
-   allow IPIP traffic. Refer to [Configuring IP-in-IP](../../usage/configuration/ip-in-ip) for more information.
+\* {% if include.orch == "OpenStack" %} If you don't use IPIP, because your
```

Comment:
The sentence is a little awkward. I think at a minimum there is one extra comma but I might suggest rephrasing it, if possible. Perhaps something like "If your compute hosts connect directly and don't use IPIP, you don't need to allow IPIP traffic." Up to you, but maybe at least delete the first comma. thanks!

Reply:
Sure, I'll switch to your wording...